src/components/thumbnail-card/ThumbnailCard.scss

```diff
@@ -13,6 +13,21 @@
     box-shadow: 0 5px 10px 0 rgba(198, 198, 198, .2);
 }
 
+// We should expand the card only when it is hovered or the action item is active
+.thumbnail-card.is-expandable {
+    &.is-action-item-active,
+    &:hover {
```

Comment:
Should these styles apply to `focus` too for accessibility?

Reply:
I don't think this card is focusable. It doesn't look like the original cards are focusable either

Reply:
Should the cards be focusable?

Reply:
I'm not sure how they're used, but I would learn towards an experience where interaction elements are accessible via keyboard and mouse.

Reply:
presumably, if one can have a menu in this card, then focusing the menu should both focus the card (bring it into view), and the menu